packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -28,51 +25,47 @@ const MainContentColumn = styled.div<
         return '1fr';
     }
   }};
-  gap: ${(props) => (props.gapSize === '10' ? '32px' : '64px')};
+  gap: ${(props) =>
+    props.gapSize === '10' ? designTokens.spacing50 : designTokens.spacing70};
   width: 100%;
 `;
 
-const RightContentMarginColumn = styled.div`
-  grid-area: right-margin;
-`;
-
-const LeftContentInnerColumn = styled.div`
+const LeftContentColumn = styled.div`
   grid-area: left-column;
 `;
 
-const RightContentInnerColumn = styled.div`
+const RightContentColumn = styled.div`
   grid-area: right-column;
   position: sticky;
```

Comment:
FYI: When using `position: sticky` I think you need to specify the `top` property otherwise it won't stick.

Reply:
Yes, I was still trying to figure it out. Thanks for the help!

Updated here: c974b238101f5b0763715709762773eb90999158